models/public/yolo-v4-tf/yolo-v4-tf.md

```diff
@@ -0,0 +1,138 @@
+# yolo-v4-tf
+
+## Use Case and High-Level Description
+
+YOLO v4 is a real-time object detection model based on ["YOLOv4: Optimal Speed and Accuracy of Object Detection"](https://arxiv.org/abs/2004.10934) paper It was implemented with Keras\* from this [repository](https://github.com/david8862/keras-YOLOv3-model-set) and converted to TensorFlow\* framework. This model was pretrained on COCO\* dataset with 80 classes.
```

Comment:
```suggestion
YOLO v4 is a real-time object detection model based on ["YOLOv4: Optimal Speed and Accuracy of Object Detection"](https://arxiv.org/abs/2004.10934) paper. It was implemented with Keras\* from this [repository](https://github.com/david8862/keras-YOLOv3-model-set) and converted to TensorFlow\* framework. This model was pretrained on COCO\* dataset with 80 classes.
```

Reply:
not sure that part about keras grammatically correct, sounds like it is link on fork of keras, not repo with model training